test/common_utils/test_case_utils.py

```diff
@@ -69,6 +69,9 @@ def skipIfNoModule(module, display_name=None):
     return unittest.skipIf(not is_module_available(module), f'"{display_name}" is not available')
 
 
+def name_func(func, _, params):
+    return f'{func.__name__}_{"_".join(str(arg) for arg in params.args)}'
+
```

Comment:
I prefer not touching `name_func` from `sox_io_backend` module, because some tests have very different way of parameterization and this `name_func` is not as generally applicable as it seems. For example, in #760 I had to use different `name_func` https://github.com/pytorch/audio/pull/760/files#diff-033abc42147aa7f1b5d3b3c8bedb2116R1. 
but if you want to move this function, please put it under `test/common_utils/parameterized_utils.py`, and then maybe rename it to something like `simple_name_func`, to be a little bit less generic.

Reply:
You can also use `lambda` in test code. 

`name_func=lambda f, _, p: f'{f.__name__}_{"_".join(str(arg) for arg in p.args)}'`